maintenance_releases.pt.Rmd

```diff
@@ -15,7 +15,7 @@ Observe que você pode simplificar o processo de atualização do NEWS e do cont
 
 - Recomendamos enfaticamente que os pacotes rOpenSci usem controle de versão semântico. Uma explicação detalhada está disponível na seção [capítulo de descrição](https://r-pkgs.org/description.html#version).
```

Comment:
Aparently, the translation of the hyperlink did not worked properly in this line. This hyperlink should point to lifecycle chapter.

```suggestion
- Recomendamos enfaticamente que os pacotes rOpenSci usem controle de versão semântico. Uma explicação detalhada está disponível no [capítulo de ciclo de vida do livro *R Packages*](https://r-pkgs.org/lifecycle.html).
```

Reply:
@pedropark99  but I see that in the documentation in English the link is the same (description)

Reply:
The rOpenSci saw this problem with the link at the original content, and they fixed too. Now, we need to fix here too.